wx/lib/agw/aui/framemanager.py

```diff
@@ -5080,6 +5083,13 @@ def ClosePane(self, pane_info):
         if to_destroy:
             to_destroy.Destroy()
 
+        # Now inform the app that we closed a pane.
+        e = AuiManagerEvent(wxEVT_AUI_PANE_CLOSED)
+        e.SetPane(pane_info)
+        e.SetCanVeto(False)
+        e.SetId(wxEVT_AUI_PANE_CLOSED)
```

Comment:
An Event's id attribute  is normally used for the id of the widget sending the event, not the event type. For example, for a `EVT_BUTTON` event the id is the value of the button's `GetId()`, not `wxEVT_BUTTON`. Also, it looks like the other AUI events are not setting the id (not great, but okay) so it's probably better to follow the existing pattern here.

Reply:
I see why i set the id: to distingish from other events. But now i know i can use the event type instead.